tests/coverage/utils.py

```diff
@@ -12,4 +12,11 @@ def _get_relpath_dict(rootpath: str, dict_to_update: dict):
     """Expects a dictionary of path strings to anything and returns an identical dictionary with the keys changed to
     relative path strings
     """
-    return {_get_relpath_str(Path(rootpath), Path(path)): lines for path, lines in dict_to_update.items()}
+    return {
+        _get_relpath_str(Path(rootpath), Path(path)): set(lines.to_sorted_list())
+        for path, lines in dict_to_update.items()
+    }
+
+
+def _get_lines_set_from_coverage_lines():
```

Comment:
It extracts the set of lines from a `CoverageLines` object. It's a convenience function which is used in tests so that we don't have to rewrite every test if the underlying coverage representation changes.